index.js

```diff
@@ -0,0 +1,54 @@
+const BASE_URL_1 = "https://jsonplaceholder.typicode.com/users";
+const BASE_URL_2 = "https://jsonplaceholder.typicode.com/todos";
+
+const dataUsers = fetch(BASE_URL_1).then((response) => response.json());
+
+const dataTodos = fetch(BASE_URL_2).then((response) => response.json());
+
+//solution number 1
+
+/* const agrigateUsersTodos = async () => {
+  try {
+    const [usersList, todosList] = await Promise.all([dataUsers, dataTodos]);
+
+    return usersList.map((user) => {
+      return {
+        ...user,
+        todos: todosList.filter(({ userId }) => userId === user.id),
+      };
+    });
+  } catch (error) {
+    console.log(error.message);
+  }
+};
+
+agrigateUsersTodos().then((data) => console.log(data)); */
+
+//solution number 2
+
+const agrigateUsersTodos = async () => {
+  try {
+    const [usersList, todosList] = await Promise.all([dataUsers, dataTodos]);
+
+    const todosMap = todosList.reduce((acc, todos) => {
+      if (acc[todos.userId]) {
+        acc[todos.userId].push(todos);
+      } else {
+        acc[todos.userId] = [];
```

Comment:
seems that you skip every first met todo, because you just create empty array without adding current item to this array